### ISSUE TITLE
Please support :Icon Shortcodes:

### ISSUE THREAD

**Issue by Hunanbean-Collective** [xhuajin/obsidian-tabs#26]
There is a plugin called Icon Shortcodes that allow you to insert certain icons between`: :`. similar to how Github allows that in comments. It would be nice if we could use that in Tab Names.

Thank you

p.s. In your examples in the description, you seem to be using Icons, so perhaps something similar already works, and i just do not know how to use it.

**Comment by dimiarbre** [comment]
I tried around a bit and indeed it would be a great addition!

One thing I found out is that I can put some standard smileys in the title if I do all the following:
* Get into the code block manually (thus not using the tabs built-in edit mode)
* Start typing the emoji in the title, such as `:smi` in my case
* Select the emoji you want to use via the autocomplete menu 
![image](https://github.com/user-attachments/assets/66027d3e-926f-476f-8d3a-b0e36f3414e3)

Doing so will, from what I understand, replace the string `:smile:` with something that is directly rendered by tabs at least. (I'm guessing their Unicode value) 

I obtain this result, which could be a temporary workaround if you don't need anything too complicated:
![image](https://github.com/user-attachments/assets/e4e7b9b6-6646-4387-9bb5-1e627c106772)
It does not seem to work for all emojis, but at least some of them do work like this. 

This is a workaround, and a bit tedious to do, but at least it works if you have simple use cases like me. 
However,  I don't know what is possible from a development point of view, I'll leave it up to the developers.
